### ISSUE TITLE
files displayed as empty in Rproject with Encoding

### ISSUE THREAD

**Issue by brry** [rstudio/rstudio#11153]
### System details

RStudio 2022.02.2+485 "Prairie Trillium" Release (8acbd38b0d4ca3c86c570cf4112a8180c48cc6fb, 2022-04-19) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36

### Steps to reproduce the problem

Create this .Rproj file: 
```
Version: 1.0

RestoreWorkspace: Default
SaveWorkspace: Default
AlwaysSaveHistory: Default

EnableCodeIndexing: Yes
UseSpacesForTab: Yes
NumSpacesForTab: 2
Encoding: ISO8859-1

RnwWeave: knitr
LaTeX: pdfLaTeX
```

### Describe the problem in detail
Open any `R`, `Rnw` or `.md` file (and probably others) and it is displayed as being empty.
My particular `.Rprofile` file _does_ get displayed with content.

Potentially related to (or even solving?) https://github.com/rstudio/rstudio/issues/2231  
I am not sure if this is a separate bug (or one at all).  

### Solution

I changed the line

```
Encoding: UTF-8
```

and everything works as expected. 

I don't know where the encoding came from, but the project is in a dropbox folder and I may have opened it on my MacBook. Dropbox has no versions tracked for this `.Rproj` file (for whatever reason) and it is in .gitignore to keep the [repo page](https://github.com/brry/hour) short, so I can't check.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@brry Thank you for raising this! I haven't been able to reproduce this yet, but I'll try a couple more things. Regarding `Encoding: ISO8859-1`, this could be if the file was first created on MacOS.

**Comment by ronblum** [comment]
Triage: Could this be related to #1623? This might be a regression.

**Comment by mikebessuille** [comment]
Duplicate of https://github.com/rstudio/rstudio/issues/11206 which is being fixed in Spotted Wakerobin (2022.06) and will be backported to Prairie Trillium.
